docs/running-tests/from-local-system.md

```diff
@@ -4,36 +4,67 @@ The tests are designed to be run from your local computer.
 
 ## System Setup
 
-The test environment requires [Python 2.7+](http://www.python.org/downloads)
-(but not Python 3.x).
+Running the tests requires `python`, `pip` and `virtualenv`, as well as updating
+the system `hosts` file.
 
-On Windows, be sure to add the Python directory (`c:\python2x`, by default) to
-your `%Path%` [Environment Variable](http://www.computerhope.com/issues/ch000549.htm),
-and read the [Windows Notes](#windows-notes) section below.
+Note that Python 2.7 is required. Using Python 3 is not supported.
 
-<!--
-  There does not appear to be a cross-platform means of installing `pip`.
-  https://github.com/web-platform-tests/wpt/pull/16670
--->
+The required setup is different depending on your operating system.
 
-Install `pip`. On many systems, this can be achieved with the command `python
--m ensurepip`. If this is not possible, use your system's package manager to
-install the `python-pip` package.
+### Linux Setup
 
-Next, install `virtualenv` using the following command:
+If not already present, use the system package manager to install `python`,
+`pip` and `virtualenv`.
+
+On Debian or Ubuntu:
+
+```bash
+sudo apt-get install python python-pip virtualenv
+```
+
+### macOS Setup
+
+The system-provided Python can be used, while `pip` and `virtualenv` can be
+installed for the user only:
+
+```bash
+python -m ensurepip --user
+export PATH="$PATH:$HOME/Library/Python/2.7/bin"
+pip install --user virtualenv
+```
+
+To make the `PATH` change persistent, add it to your `~/.profile` file.
```

Comment:
```suggestion
To make the `PATH` change persistent, add it to your `~/.bash_profile` file or
wherever you currently set your PATH.
```

Reply:
This is tricky. Bash looks for a profile in this order: `~/.bash_profile`, `~/.bash_login`, and `~/.profile`, and stops at the first found. Homebrew apparently prefers to write to `.bash_profile`, which would shadow `.profile`. This has bitten me once before...